### PR TITLE
Add test with Revault unvault_descriptor without cosigners

### PR DESCRIPTION
**Pull request by edouardparis**

### CODE REVIEW

**Comment by edouardparis** [comment]
passphrase: 
`rapid veteran belt horse evidence wine rabbit price protect foam summer excuse`

```
addwallet deposit_wallet&wsh(multi(2,tpubD6NzVbkrYhZ4WmzFjvQrp7sDa4ECUxTi9oby8K4FZkd3XCBtEdKwUiQyYJaxiJo5y42gyDWEczrFpozEjeLxMPxjf2WtkfcbpUdfvNnozWF/*,tpubD6NzVbkrYhZ4XyJXPpnkwCpTazWgerTFgXLtVehbPyoNKVFfPgXRcoxLGupEES1tSteVGsJon85AxEzGyWVSxm8LX8bdZsz87GWt585X2wf/*))#36w5x8qy
addwallet unvault_wallet&wsh(andor(thresh(1,pk(tpubD6NzVbkrYhZ4XcB3kRJVob8bmjMvA2zBuagidVzh7ASY5FyAEtq4nTzx9wHYu5XDQAg7vdFNiF6yX38kTCK8zjVVmFTiQR2YKAqZBTGjnoD/*)),older(10),thresh(2,pkh(tpubD6NzVbkrYhZ4WmzFjvQrp7sDa4ECUxTi9oby8K4FZkd3XCBtEdKwUiQyYJaxiJo5y42gyDWEczrFpozEjeLxMPxjf2WtkfcbpUdfvNnozWF/*),a:pkh(tpubD6NzVbkrYhZ4XyJXPpnkwCpTazWgerTFgXLtVehbPyoNKVFfPgXRcoxLGupEES1tSteVGsJon85AxEzGyWVSxm8LX8bdZsz87GWt585X2wf/*))))#u70luyrn
addwallet cpfp_wallet&wsh(multi(1,tpubD6NzVbkrYhZ4XcB3kRJVob8bmjMvA2zBuagidVzh7ASY5FyAEtq4nTzx9wHYu5XDQAg7vdFNiF6yX38kTCK8zjVVmFTiQR2YKAqZBTGjnoD/*))#zafxqvr4

```

Error with example of an unvault transaction:
```
deposit_descriptor -> unvault_descriptor
                              -> cpfp_descriptor
```
```
cHNidP8BAIkCAAAAAXp7M2qYCFi5WiF5cDMWwHwbXsjCDUHCmfRhX7PtvY+yAAAAAAD9////AliS9wcAAAAAIgAgI88K3dsiL1U8Vmg3JLhg6gUnFCXGvTkK+781DS0uewYwdQAAAAAAACIAIAuKEfojuBgzEohoFjUx1ZMcWbrbErz/Lrfx5VlmmXy5AAAAAAABASugGfgHAAAAACIAICTJOG78TIre8heykxyu0c9IkfSXcFJq1XA9aJO0kQLOAQMEAQAAAAEFR1IhA2FPUub/h0vtcc2t6wynvtrOcbQPPoBFmCyCIvzy5ET7IQK+zF69BknINpIs6KcrVM+dvcnX3lYWPc9wY2izta3jCVKuIgYCvsxevQZJyDaSLOinK1TPnb3J195WFj3PcGNos7Wt4wkI5wT/vgAAAAAiBgNhT1Lm/4dL7XHNresMp77aznG0Dz6ARZgsgiL88uRE+wiKZPKpAAAAAAAiAgKm242c21PacXWunfRWSZwk7VablpGPi7Vg8DQ4M9cPCQh+CFEoAAAAACICAr7MXr0GScg2kizopytUz529ydfeVhY9z3BjaLO1reMJCOcE/74AAAAAIgIDYU9S5v+HS+1xza3rDKe+2s5xtA8+gEWYLIIi/PLkRPsIimTyqQAAAAAAIgICptuNnNtT2nF1rp30VkmcJO1Wm5aRj4u1YPA0ODPXDwkIfghRKAAAAAAA
```

**Comment by darosior** [comment]
FWIW, the C++ implementation gives `bcrt1qy08s4hwmygh420zkdqmjfwrqagzjw9p9c67njzhmhu6s6tfw0vrq9sfn4y` for the derivation of 
```
wsh(andor(thresh(1,pk(tpubD6NzVbkrYhZ4XcB3kRJVob8bmjMvA2zBuagidVzh7ASY5FyAEtq4nTzx9wHYu5XDQAg7vdFNiF6yX38kTCK8zjVVmFTiQR2YKAqZBTGjnoD/*)),older(10),thresh(2,pkh(tpubD6NzVbkrYhZ4WmzFjvQrp7sDa4ECUxTi9oby8K4FZkd3XCBtEdKwUiQyYJaxiJo5y42gyDWEczrFpozEjeLxMPxjf2WtkfcbpUdfvNnozWF/*),a:pkh(tpubD6NzVbkrYhZ4XyJXPpnkwCpTazWgerTFgXLtVehbPyoNKVFfPgXRcoxLGupEES1tSteVGsJon85AxEzGyWVSxm8LX8bdZsz87GWt585X2wf/*))))#u70luyrn
```
at index 0.

**Comment by edouardparis** [comment]
Specter-diy does not have the same address for index 0
2021-12-20T15:21:04,558334484+01:00
2021-12-20T15:21:25,670195456+01:00

File: test/tests/test_revault.py [hunk 1]
```diff
@@ -9,11 +9,14 @@
...
+    # unvault_descriptor without cosigners and using tpub
+    "unvault_descriptor_without_cosigs&wsh(andor(thresh(1,pk(tpubD6NzVbkrYhZ4XcB3kRJVob8bmjMvA2zBuagidVzh7ASY5FyAEtq4nTzx9wHYu5XDQAg7vdFNiF6yX38kTCK8zjVVmFTiQR2YKAqZBTGjnoD/*)),older(10),thresh(2,pkh(tpubD6NzVbkrYhZ4WmzFjvQrp7sDa4ECUxTi9oby8K4FZkd3XCBtEdKwUiQyYJaxiJo5y42gyDWEczrFpozEjeLxMPxjf2WtkfcbpUdfvNnozWF/*),a:pkh(tpubD6NzVbkrYhZ4XyJXPpnkwCpTazWgerTFgXLtVehbPyoNKVFfPgXRcoxLGupEES1tSteVGsJon85AxEzGyWVSxm8LX8bdZsz87GWt585X2wf/*))))#u70luyrn",
```
Comment (stepansnigirev) [inline test/tests/test_revault.py:13]: Tests are using mainnet xpubs here. Probably better to use testnet and tpubs, but for now let's use xpubs. I'll update it to use testnet later.
If you update f469-disco submodule to master test will pass. I fixed the thresh issue.

```suggestion
    # unvault_descriptor without cosigners and using xpub
    "unvault_descriptor_without_cosigs&wsh(andor(thresh(1,pk(xpub661MyMwAqRbcFozCJEKQbfoEScGGAeV8Mx6RGwwYmFgeKyJTAfyye2L7uuCmNaZycG9DK3jWB9Gvpre94LTu7J3CheiQitNJjDF9RMtT5Kt/*)),older(10),thresh(2,pkh(xpub661MyMwAqRbcEyoQHjRmcCXrEw8YVZxecB1fmm17Dqs9muXBAQUrLGk9JGWBBoqrB9VnMdzN5u2D8dVdLnViTxWSbRmb58xNEX3GARisZBx/*),a:pkh(xpub661MyMwAqRbcGB7fwdofjHV6FsR2fTxC8tkb96eT453UaCaxKTgLUNHW2sjShw4eez7afHnwF2F8G4VfaeeD5Kg3TXrKtMKtXJvUK3Jh3uE/*))))",
```

Comment (edouardparis) [reply test/tests/test_revault.py]: Ok, thank you for the suggestion and the fix

**Pull request merged**